sdk/identity/identity/package.json

```diff
@@ -72,7 +72,7 @@
   "sideEffects": false,
   "dependencies": {
     "@azure/core-http": "^1.0.0",
-    "@azure/core-tracing": "1.0.0-preview.6",
```

Comment:
In general we shouldn't depend on preview packages from GA package releases. Is the plan to GA tracing and update before we ship the next version of this package that depends on tracing?

Reply:
let's continue this discussion in #6371